source/docs/articles/users/access-management.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Access Management for Site Owners and Administrators
+description: Learn how to remove a team member or outside resource from a site.
+keywords: remove team member, delete team member, delete user
+---
+
+When a resource leaves the company or project, it is important to immediately remove them from the team  so that they no longer have access to make changes to your site. Pantheon makes it easy to manage your team, so you do not have to do any work within the OS or change users and passwords in the database.
+
+If you suspect the team member made code changes after leaving, go to the Code tab and review the Git history in the commit log.
+
+## Remove a Team Member from a Site
```

Comment:
How about linking to existing documentation? https://pantheon.io/docs/articles/sites/team-management/#remove-a-team-member